test/e2e/egress_test.go

```diff
@@ -168,13 +168,25 @@ ip netns exec %[1]s /agnhost netexec
 	assertClientIP(localPod, egressNodeIP)
 	assertClientIP(remotePod, egressNodeIP)
 
+	err := wait.Poll(time.Millisecond*100, time.Second, func() (bool, error) {
+		egress, err := data.crdClient.CrdV1alpha2().Egresses().Get(context.TODO(), egress.Name, metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		return egress.Status.EgressNode == egressNode, nil
+	})
+	assert.NoError(t, err, "Egress failed to reach expected status")
+
+	egress, err = data.crdClient.CrdV1alpha2().Egresses().Get(context.TODO(), egress.Name, metav1.GetOptions{})
```

Comment:
Why it needs to get it again after the above loop has verified its status?
You can declare egress out of the loop.

Reply:
done